### ISSUE TITLE
Add term thickened cortex of the clavicle

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10454]
Preferred term label:
Thickened cortex of the clavicle

Synonyms
Broad cortex of clavicle- Cortical thickening of the clavicle- Thickened cortices of clavicle

Definition (free text, please give PubMed ID)
Abnormal thickening of the cortex of the clavicle. PMID24389367

Parent term (use hpo.jax.org/app)
Thickened cortex of long bones HP:0000935
Abnormal clavicle morphology HP:0000889

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Caffey Disease (omim114000), for example: PMID24390061

Your nano-attribution (ORCID)
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000936
- label: Thickened cortex of the clavicle
- parents: [Abnormal clavicle morphology (HP:0000889)](https://hpo.jax.org/app/browse/term/HP:0000889)
- pmid: PMID:24389367|PMID:24390061
- def: Abnormal thickening of the cortex of the clavicle.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10454
- synonyms: Broad cortex of clavicle(NONE)|Thickened cortices of clavicle(NONE)|Cortical thickening of the clavicle(NONE)
